doc/api/modules.md

```diff
@@ -425,6 +426,30 @@ These are mostly for historic reasons.  **You are highly encouraged
 to place your dependencies locally in `node_modules` folders.**  They
 will be loaded faster, and more reliably.
 
+## The module wrapper
+
+<!-- type=misc -->
+
+Before Node.js hands off your module to be executed by V8, it wraps your code
```

Comment:
I wouldn’t reference V8 here directly, as this does not depend on the specific engine that’s being used.